README.md

```diff
@@ -42,33 +42,39 @@ Here are some pointers to get started with Multipass. For a more comprehensive l
 $ multipass find
 
 Image                       Aliases           Version          Description
-core                        core16            20200213         Ubuntu Core 16
-core18                                        20200210         Ubuntu Core 18
-16.04                       xenial            20200721         Ubuntu 16.04 LTS
-18.04                       bionic,lts        20200717         Ubuntu 18.04 LTS
-20.04                       focal             20200720         Ubuntu 20.04 LTS
-daily:20.10                 devel,groovy      20200721         Ubuntu 20.10
+20.04                       focal             20240731         Ubuntu 20.04 LTS
+22.04                       jammy             20240808         Ubuntu 22.04 LTS
+24.04                       noble,lts         20240806         Ubuntu 24.04 LTS
+
+Blueprint                   Aliases           Version          Description
+anbox-cloud-appliance                         latest           Anbox Cloud Appliance
+charm-dev                                     latest           A development and testing environment for charmers
+docker                                        0.4              A Docker environment with Portainer and related tools
+jellyfin                                      latest           Jellyfin is a Free Software Media System that puts you in control of managing and streaming your media.
+minikube                                      latest           minikube is local Kubernetes
+ros-noetic                                    0.1              A development and testing environment for ROS Noetic.
+ros2-humble                                   0.1              A development and testing environment for ROS 2 Humble.
 ```
 
 ## Launch a fresh instance of the current Ubuntu LTS
 
 ```
-$ multipass launch ubuntu
+$ multipass launch lts
 
-Launching dancing-chipmunk...
-Downloading Ubuntu 18.04 LTS..........
-Launched: dancing chipmunk
+Launched: dancing-chipmunk
 ```
 
 ## Check out the running instances
 
 ```
 $ multipass list
 
-Name                    State             IPv4             Release
-dancing-chipmunk        RUNNING           10.125.174.247   Ubuntu 18.04 LTS
-live-naiad              RUNNING           10.125.174.243   Ubuntu 18.04 LTS
-snapcraft-asciinema     STOPPED           --               Ubuntu Snapcraft builder for Core 18
+Name                    State             IPv4             Image
+dancing-chipmunk        Running           192.168.64.8     Ubuntu 24.04 LTS
```

Comment:
Is the `dancing-chipmunk` the result instance of `multipass launch lts`? If so, they it should only have one IP address, right?

Reply:
The output here is copy-pasted from the output of `multipass list` on my Mac, with the only difference that I kept the instance name "dancing-chipmunk" that was originally in the README because I like it :) Although this output is only provided as an example, I guess it makes more sense to only show one IP address, since in these instruction the instance was freshly created. I'll fix it! Thanks for catching it.